zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs

```diff
@@ -422,6 +422,24 @@ pub async fn test_responses<State, ReadState>(
         .expect("unexpected error in getdifficulty RPC call");
 
     snapshot_rpc_getdifficulty(get_difficulty, &settings);
+
+    let ua1 = String::from("u1l8xunezsvhq8fgzfl7404m450nwnd76zshscn6nfys7vyz2ywyh4cc5daaq0c7q2su5lqfh23sp7fkf3kt27ve5948mzpfdvckzaect2jtte308mkwlycj2u0eac077wu70vqcetkxf");
+    let z_list_unified_receivers =
+        tokio::spawn(get_block_template_rpc.z_list_unified_receivers(ua1))
+            .await
+            .expect("unexpected panic in z_list_unified_receivers RPC task")
+            .expect("unexpected error in z_list_unified_receivers RPC call");
+
+    snapshot_rpc_z_listunifiedreceivers("ua1", z_list_unified_receivers, &settings);
+
+    let ua2 = String::from("u1uf4qsmh037x2jp6k042h9d2w22wfp39y9cqdf8kcg0gqnkma2gf4g80nucnfeyde8ev7a6kf0029gnwqsgadvaye9740gzzpmr67nfkjjvzef7rkwqunqga4u4jges4tgptcju5ysd0");
```

Comment:
Can you do a new `zcash-rpc-diff` with these test vectors?

We haven't checked that our p2sh matches `zcashd` yet.

Reply:
```
$ ./zebra-utils/zcash-rpc-diff 6666 z_listunifiedreceivers u1uf4qsmh037x2jp6k042h9d2w22wfp39y9cqdf8kcg0gqnkma2gf4g80nucnfeyde8ev7a6kf0029gnwqsgadvaye9740gzzpmr67nfkjjvzef7rkwqunqga4u4jges4tgptcju5ysd0
Checking first node release info...
Checking second node release info...
Connected to zebrad (port 6666) and zcashd (/media/oxarbitrage/4eb53770-4e4f-4d80-a830-0914f3f5b89a/zcash/zcash/src/zcash-cli zcash.conf port).

Checking zebrad network and tip height...
Checking zcashd network and tip height...

WARNING: comparing RPC responses from different heights:
zebrad is at: 1990596
zcashd is at: 1990075

Request:
z_listunifiedreceivers u1uf4qsmh037x2jp6k042h9d2w22wfp39y9cqdf8kcg0gqnkma2gf4g80nucnfeyde8ev7a6kf0029gnwqsgadvaye9740gzzpmr67nfkjjvzef7rkwqunqga4u4jges4tgptcju5ysd0

Querying zebrad main chain at height >=1990596...

real	0m0,002s
user	0m0,002s
sys	0m0,000s

Querying zcashd main chain at height >=1990075...

real	0m0,001s
user	0m0,000s
sys	0m0,001s


Response diff between zebrad and zcashd:
RPC responses were identical

/tmp/tmp.48R7ON9siI.rpc-diff/zebrad-main-1990596-z_listunifiedreceivers.json:
{
  "orchard": "u10c5q7qkhu6f0ktaz7jqu4sfsujg0gpsglzudmy982mku7t0uma52jmsaz8h24a3wa7p0jwtsjqt8shpg25cvyexzlsw3jtdz4v6w70lv",
  "p2sh": "t3dvVE3SQEi7kqNzwrfNePxZ1d4hUyztBA1"
}
$ 
```